lib0/src/any.rs

```diff
@@ -140,4 +135,33 @@ impl<T> Into<Any> for HashMap<String, T> where T: Into<Any> {
         }
         Any::Map(map)
     }
+}
+
+impl TryInto<Any> for u64 {
+    type Error = &'static str;
+
+    fn try_into(self) -> Result<Any, Self::Error> {
+        if self <= (1 << 53) {
```

Comment:
The largest integer that a double can represent is `2^53-1`. So this needs to be `<`.